### PR TITLE
chore: bump gradle from 3.3.2 to 3.4.1

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #1768

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Mon Jan 21 18:56:58 IST 2019
+#Thu May 16 13:42:01 IST 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.1.1-all.zip
```
Comment (ShridharGoel) [inline gradle/wrapper/gradle-wrapper.properties]: Change to 5.4.1

Comment (liveHarshit) [reply gradle/wrapper/gradle-wrapper.properties]: Why?

Comment (ShridharGoel) [reply gradle/wrapper/gradle-wrapper.properties]: Because it's the latest version

Comment (liveHarshit) [reply gradle/wrapper/gradle-wrapper.properties]: Please go through Travis logs - https://travis-ci.org/fossasia/open-event-android/builds/533186940?utm_source=github_status&utm_medium=notification

Comment (ShridharGoel) [reply gradle/wrapper/gradle-wrapper.properties]: `Minimum supported Gradle version is 5.1.1`

The minimum supported version is 5.1.1. Versions after that can also be used. So shouldn't we use the latest available version?

**Comment by liveHarshit** [comment]
@iamareebjamal Gradle version 3.4 requires Android Studio 3.4 or newer. Is it okay?

File: app/src/main/java/org/fossasia/openevent/general/search/PlaceSuggestionsAdapter.kt [hunk 2]
```diff
@@ -36,7 +36,7 @@ class PlaceSuggestionsAdapter :
         }
 
         override fun areContentsTheSame(oldItem: CarmenFeature, newItem: CarmenFeature): Boolean {
-            return oldItem == newItem
+            return oldItem.equals(newItem)
```
Comment (liveHarshit) [inline app/src/main/java/org/fossasia/openevent/general/search/PlaceSuggestionsAdapter.kt]: image

**Comment by liveHarshit** [comment]
Updated. Build successful.

**Pull request merged**